src/main/scala/amailp/intellij/robot/psi/KeywordDefinition.scala

```diff
@@ -2,8 +2,11 @@ package amailp.intellij.robot.psi
 
 import amailp.intellij.robot.ast
 import com.intellij.lang.ASTNode
+import com.intellij.openapi.project.Project
 import com.intellij.psi._
 import com.intellij.psi.util.PsiTreeUtil
+import com.jetbrains.python.psi.{PyFunction, PyFile}
```

Comment:
The plugin should work even without the python plugin installed (for java users for example). In this case I guess they would get a ClassDefNotFound exception that is not what should happen. If you look at the moment the com.jetbrains.python.* classes are only loaded when the optional python plugin is present via an optional plugin desctriptor (META-INF/robot-python.xml)